### PR TITLE
Direct livechat.ly.fish requests to ui pods

### PR DESCRIPTION
**Pull request by joshbwlng**
Change-type: patch
Signed-off-by: Josh Bowling <josh@balena.io>

***

Update deploy templates to reflect the recent work of serving livechat from the ui containers. This PR assumes that we no longer need the livechat containers at all and should instead direct all requests to `livechat.ly.fish` to the ui containers.

### CODE REVIEW

File: deploy-templates/jellyfish-product/product/deploy/kubernetes/generators/jellyfish-ui-ingress.tpl.yml [hunk 1]
```diff
@@ -45,3 +45,14 @@ spec:
             name: jellyfish-ui
             port:
               number: 80
+  - host: '{{{JELLYFISH_LIVECHAT_HOST}}}'
```
Comment (joshbwlng) [inline deploy-templates/jellyfish-product/product/deploy/kubernetes/generators/jellyfish-ui-ingress.tpl.yml:48]: Ah, actually, we probably would instead want to keep the livechat ingress, but point it to the the ui pods

**Review by karaxuna — APPROVED** [review]
LGTM

**Comment by karaxuna** [comment]
@balena-ci rebase

**Comment by joshbwlng** [comment]
@karaxuna Thanks for the review, but I'd like to have @ab77 and/or @cmfcruz take a look before deploying. I haven't tested these configs and there is a chance that it could be botch deployment.

**Comment by joshbwlng** [comment]
@balena-ci rebase

**Pull request merged**